extra_data/cli/check_readable.py

```diff
@@ -0,0 +1,141 @@
+
+import subprocess
+import sys
+from argparse import ArgumentParser
+from multiprocessing.pool import ThreadPool
+from os import cpu_count
+from pathlib import Path
+from time import monotonic
+
+from extra_data import FileAccess
+from extra_data.locality import get_locality
+
+
+def check_access(path):
+    print('any access hangs', flush=True)
+
+    start = monotonic()
+    locality = get_locality(path)
+
+    if locality == 'UNAVAILABLE':
+        print('dCache reports unavailable', flush=True)
```

Comment:
Is the explicit flush necessary here? Libc's `exit()` guarantees that stdio streams are flushed, which I believe is what Python will call upon a normal exit.

Reply:
Happy to add it for symmetry, but at that point the process actually will exit gracefully 🙃 

The `flush=True` is essentially needed as long as you're still doing calls that may hang.